pennylane_lightning/core/_serialize.py

```diff
@@ -54,12 +54,18 @@ class QuantumScriptSerializer:
     use_csingle (bool): whether to use np.complex64 instead of np.complex128
     use_mpi (bool, optional): If using MPI to accelerate calculation. Defaults to False.
     split_obs (Union[bool, int], optional): If splitting the observables in a list. Defaults to False.
+    tensor_backend (str): If using lightning.tensor and select the TensorNetwork backend, mps or exact. Default to mps
```

Comment:
in the arguments the default is "", should we put as mps?

Reply:
nop the default should be empty. I updated the docstring